From cb833dcade916ecb281b9eb9ea8f580ccfd64569 Mon Sep 17 00:00:00 2001
From: Chippy <chippy@classictetris.net>
Date: Mon, 11 Sep 2023 14:41:48 +0200
Subject: [PATCH 03/13] This patch could not be applied as the entire code to
 patch is now missing from the target file old-configure.in

---
 .../patches/{ => moved_by_mozilla}/fix-data-reporting-check.patch | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename data/patches/{ => moved_by_mozilla}/fix-data-reporting-check.patch (100%)

diff --git a/data/patches/fix-data-reporting-check.patch b/data/patches/moved_by_mozilla/fix-data-reporting-check.patch
similarity index 100%
rename from data/patches/fix-data-reporting-check.patch
rename to data/patches/moved_by_mozilla/fix-data-reporting-check.patch
-- 
2.40.1

